layers/generate_layers.py

```diff
@@ -467,29 +505,60 @@ def summarizeCarAvailability(row):
     }
 
 
+def generateGeojsonFromOSMFile(osmFilePath, outputFilepath):
+    run(
+        [
+            "osmium",
+            "export",
+            osmFilePath,
+            "-o",
+            outputFilepath,
+        ]
+    )
+
+
+def ensureEmptyTempDirectoryExists(directoryName):
+    if os.path.isdir(directoryName):
+        run(["rm", "-r", directoryName])
+    os.makedirs(directoryName, exist_ok=True)
+
+
 def run(args):
     print(">", " ".join(args))
     subprocess.run(args, check=True)
 
 
 # For each GeoJSON feature, keep only the name attribute. Overwrites the given file.
-def remove_extra_properties(path):
-    print(f"Removing extra properties from {path}")
+def removeNonNameProperties(path):
```

Comment:
This doesn't seem any more clear than just doing `cleanUpGeojson(path, ["name"])` at the call-sites